pkg/controller/nodenetworkstate/nodenetworkstate_controller.go

```diff
@@ -36,8 +41,22 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	forThisNode := predicate.Funcs{
+		CreateFunc: func(createEvent event.CreateEvent) bool {
+			return nmstate.EventIsForThisNode(createEvent.Meta)
+		},
+		DeleteFunc: func(deleteEvent event.DeleteEvent) bool {
+			return nmstate.EventIsForThisNode(deleteEvent.Meta)
```

Comment:
What about ignoring it for now? There is no point on updating content of a NodeNetworkState that has been marked to be removed

Reply:
Done.